app/models/bgeigie_import.rb

```diff
@@ -15,7 +15,7 @@ class BgeigieImport < MeasurementImport # rubocop:disable Metrics/ClassLength
   has_many :bgeigie_logs, dependent: :delete_all
   has_many :uploader_contact_histories, dependent: :delete_all
 
-  scope :newest, -> { order('created_at DESC') }
+  scope :newest, ->(*) { order(created_at: :desc) }
```

Comment:
First time I've seen `(*)` - why is is required just on this scope?

Reply:
Because I wanted to use `:newest` as default scope, and I followed [this documentation](
https://github.com/plataformatec/has_scope#apply-scope-on-every-request)